crates/codegen/runtime/cargo/wasm/src/runtime/interface/cst.wit.jinja2

```diff
@@ -11,7 +11,11 @@ interface cst {
             stub3,
         {%- else %}
             {%- for variant in model.kinds.nonterminal_kinds %}
-                {{ variant | wit_case }},
+                /// This kind represents a `{{ variant.id }}` node, with the following structure: 
+                {%- for line in variant.documentation | split(pat="\n") %}
+                /// {{ line }}
+                {%- endfor %}
```

Comment:
here + below: I suggest wrapping them in a code block, to get syntax highlighting there:

```suggestion
                /// This kind represents a `{{ variant.id }}` node, with the following structure:
                ///
                /// ```ebnf
                {%- for line in variant.documentation | split(pat="\n") %}
                /// {{ line }}
                {%- endfor %}
                /// ```
```

Reply:
I think the Rust ones can benefit from this as well:

- `crates/solidity/outputs/cargo/crate/src/generated/cst/generated/terminal_kind.rs`
- `crates/solidity/outputs/cargo/crate/src/generated/cst/generated/nonterminal_kind.rs`